app/test/frontend/golden/flutter_packages_index_page2.html

```diff
@@ -114,20 +114,22 @@
 
 <div class="pagination">
   <ul>
-      <li class="">
-        <a href="/packages?page=1">
+      <li class=" ">
```

Comment:
Why the extra whitespace?

Reply:
Simpler rendering has its downsizes. Do you prefer the tradeoff for the better HTML output?